lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -569,6 +569,13 @@ module.exports = class MethodMappings {
           return element.isDisplayed();
         },
 
+        isElementPresent(webElementOrId) {
+          const element = this.getWebElement(webElementOrId);
+          const value = element.findElement();
```

Comment:
This is an invalid call, `findElement` should always contain an argument.

Reply:
I will re investigate this problem tomorrow